R/build-home-authors.R

```diff
@@ -105,22 +105,49 @@ author_list <- function(x, authors_info, comment = FALSE) {
   roles <- paste0(role_lookup[x$role], collapse = ", ")
   substr(roles, 1, 1) <- toupper(substr(roles, 1, 1))
 
+  orcid <- x$comment %>% purrr::pluck('ORCID')
+  x <- remove_orcid_comment(x)
```

Comment:
If the author info is regarded as pretty ephemeral or relevant in a small scope, maybe it's OK for the ORCID to be present in two places and you don't have to update the comment?

I note that it requires some effort to uphold this: `comment` is either `NULL` or a character vector with length > 0. Possibly not worth it.